rllib/policy/tf_policy.py

```diff
@@ -253,6 +253,13 @@ def compute_actions(self,
                         timestep=None,
                         **kwargs):
         explore = explore if explore is not None else self.config["explore"]
+
+        # Call the exploration before_compute_actions hook.
+        self.exploration.before_compute_actions(
+            obs_batch=obs_batch,
+            timestep=self.global_timestep,
+            tf_sess=self.get_session())
```

Comment:
Should also do in _build_compute_actions()

Reply:
Done